thread/src/components/Logo.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, Image } from 'react-native';
+import { Text } from 'native-base';
+import styled from "styled-components";
+
+
+const StyledView = styled.View`
+
+  padding-left:70px;
```

Comment:
this positioning does not work for small screens as seen on slack image. Please use flex to center.